independent-publisher-connectors/Microsoft 365 Service Health/apiProperties.json

```diff
@@ -0,0 +1,51 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "aad",
+          "clientId": "00000000-0000-0000-0000-000000000000",
```

Comment:
@techienickb You can just provide dummy string as value, like below: 
`"clientId": "[[DUMMY]]"`